.ci/deps.sh

```diff
@@ -14,7 +14,7 @@ sudo add-apt-repository -y ppa:staticfloat/juliareleases
 sudo add-apt-repository -y ppa:staticfloat/julia-deps
```

Comment:
is this fixing an issue? if so, please link it with "Fixes issue_url"

also change commit message please, such as: "ci: Upgrade ..."
and body should explain why: "When initialising Julia bla bla, an error was shown. This helps prevent that error from coming back up."